vms/platformvm/service_test.go

```diff
@@ -73,7 +73,7 @@ var (
 )
 
 func defaultService(t *testing.T) (*Service, *mutableSharedMemory) {
-	vm, _, mutableSharedMemory := defaultVM(t)
+	vm, _, mutableSharedMemory := defaultVM(t, latestFork)
```

Comment:
P-chain UTs do not handle correctly forks at the moment.
Where is a bunch of PRs geared to fix this (see https://github.com/ava-labs/avalanchego/pull/2169).
While we wait to have them reviewed and merged, this is the minimal amount of changes I needed to have this PR tested meaningfully.